index.html

```diff
@@ -16,25 +16,54 @@
 <body>
```

Comment:
Everything is according to the wireframe! But please do not disable eslint rules and resolve the issues instead.

Reply:
luxon.js has hundreds of linter errors so we disabled it.